Prox/Prox/Data/PlacesController.swift

```diff
@@ -30,32 +30,22 @@ class PlacesProvider {
         return RemoteConfigKeys.searchRadiusInKm.value
     }()
 
-    fileprivate var displayedPlaces = [Place]() {
-        didSet {
-            var placesMap = [String: Int]()
-            for (index, place) in displayedPlaces.enumerated() {
-                placesMap[place.id] = index
-            }
-            self.placeKeyMap = placesMap
-        }
-    }
-    fileprivate var placeKeyMap = [String: Int]()
+    private var allPlaces = [Place]()
 
+    fileprivate var displayedPlaces = [Place]()
```

Comment:
I wonder if there would be a benefit separating the data (allPlaces) from the representation/transformations (displayedPlaces) in two separate classes. So we have a PlacesFetcher which does the fetch, then we have a PlacesPresenter which can sort, filter, etc. data from the PlacesFetcher. _Something something single responsibility principle..._ :P